Source/Core/DolphinQt/Debugger/MemoryWidget.cpp

```diff
@@ -689,6 +693,53 @@ void MemoryWidget::OnSetValue()
   Update();
 }
 
+void MemoryWidget::OnSetValueFromFile()
+{
+  if (!Core::IsRunning())
+    return;
+
+  auto target_addr = GetTargetAddress();
+
+  if (!target_addr.is_good_address)
+  {
+    ModalMessageBox::critical(this, tr("Error"), tr("Bad address provided."));
+    return;
+  }
+
+  if (!target_addr.is_good_offset)
+  {
+    ModalMessageBox::critical(this, tr("Error"), tr("Bad offset provided."));
+    return;
+  }
+
+  QString path = QFileDialog::getOpenFileName(this, tr("Select a file"), QDir::currentPath(),
+                                              tr("All files (*.*)"));
```

Comment:
See my comment in the other conversation. `(*.*)` should be just `(*)`, otherwise you're excluding files without an extension.